site/_posts/2020-03-31-Announcing-Jansa-Alpha.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Announcing Jansa Alpha Release
+author: chessbyte
+date: 2020-03-31 19:12:13 UTC
+comments: true
+published: true
+tags: releases announcements
+---
+
+We have just built Jansa Alpha. This release has 2 major changes that are included in the [roadmap](https://github.com/orgs/ManageIQ/projects/13#column-7337345)
+
+- Podification
```

Comment:
Perhaps something like:

- Podification - A new release format for running ManageIQ on Kubernetes and OpenShift.  This is the culmination of the rearchitecture we started almost 2 years ago.  While we've had a "podified" version for some time, it was structurally different and not really supported nor advertised, being more of a stepping stone to this new format.